Description: Debian spesific patch, to use DejaVuSansMono.ttf,
 provided by the ttf-dejavu package, instead of the bundled 
 VeraMono.ttf font which is included in muinin source.
Origin: other
Bug-Debian: http://bugs.debian.org/548508
Forwarded: no
Author: Tom Feiner <feiner.tom@gmail.com>
Last-Update: 2009-10-30
Index: munin-1.4.0/master/lib/Munin/Master/GraphOld.pm
===================================================================
--- munin-1.4.0/master/lib/Munin/Master/GraphOld.pm	(revision 2772)
+++ munin-1.4.0/master/lib/Munin/Master/GraphOld.pm	(working copy)
@@ -1094,12 +1094,12 @@
                 push(@complete, '--font', 'LEGEND:7:monospace');
             }
             else {
-                push(@complete, '--font', 'LEGEND:7:$libdir/VeraMono.ttf');
+                push(@complete, '--font', 'LEGEND:7:/usr/share/fonts/truetype/ttf-dejavu/DejaVuSansMono.ttf');
             }
 
             push(@complete,
-                '--font', 'UNIT:7:$libdir/VeraMono.ttf',
-                '--font', 'AXIS:7:$libdir/VeraMono.ttf');
+                '--font', 'UNIT:7:/usr/share/fonts/truetype/ttf-dejavu/DejaVuSansMono.ttf',
+                '--font', 'AXIS:7:/usr/share/fonts/truetype/ttf-dejavu/DejaVuSansMono.ttf');
         }
 
         push(@complete, '-W', $watermark) if $RRDs::VERSION >= 1.2;
